### ISSUE TITLE
setup.py fails on importing pypandoc

### ISSUE THREAD

**Issue by charisz** [alecthomas/voluptuous#354]
version 0.11.3

Issue:
Calling setup.py fails when trying to import `pypandoc`. However it is not a dependency of voluptuous, at least it is absent from dependencies.

Traceback:
```
Collecting voluptuous==0.11.3
  Using cached https://files.pythonhosted.org/packages/1e/e1/d4ee28be80c22d17cf2a085582fd3ced2825731a4ea53fc42ad6da920725/voluptuous-0.11.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-2wichjm5/voluptuous/setup.py", line 12, in <module>
        import pypandoc
    ImportError: No module named 'pypandoc'
```

Reproduce:
pip install voluptuous==0.11.3 --no-binary :all:

Note:
From wheel I can install it.

**Comment by svisser** [comment]
Hmm, it appears the `setup.py` in the `.tar.gz` file doesn't have the try / except as in the repo:

```python
try:
    from setuptools import setup
except ImportError:
    from distutils.core import setup

import sys
import os
import atexit
sys.path.insert(0, '.')
version = __import__('voluptuous').__version__

import pypandoc
long_description = pypandoc.convert('README.md', 'rst')
with open('README.rst', 'wb') as f:
    f.write(long_description.encode('utf-8'))
atexit.register(lambda: os.unlink('README.rst'))

description = long_description.splitlines()[0].strip()


setup(
    name='voluptuous',
    url='https://github.com/alecthomas/voluptuous',
    download_url='https://pypi.python.org/pypi/voluptuous',
    version=version,
    description=description,
    long_description=long_description,
    license='BSD',
    platforms=['any'],
    packages=['voluptuous'],
    author='Alec Thomas',
    author_email='alec@swapoff.org',
    classifiers=[
        'Development Status :: 5 - Production/Stable',
        'Intended Audience :: Developers',
        'License :: OSI Approved :: BSD License',
        'Operating System :: OS Independent',
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.6',
        'Programming Language :: Python :: 3.7',
    ]
)
```

**Comment by charisz** [comment]
Yes, because the code under tag 0.11.3 doesn't have the try/except block as weel.

**Comment by svisser** [comment]
Oh I see. Looks like https://github.com/alecthomas/voluptuous/commit/7bcc56cbfff29fe3c10268c99b4b9c598f74fc4d needs to be reverted as `pypandoc` should remain optional?

**Comment by alecthomas** [comment]
Mmmm, whoops. This was a misguided attempt to avoid pushing a release without RsTifying the docs first. I will revert it and push a new release.

**Comment by alecthomas** [comment]
Fixed in d1ec13e7575d7a25fc11ce80d72885c55867184a.

Looks like they added Markdown support woo!
